src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java

```diff
@@ -85,6 +81,14 @@ public static Schema getSchema(SnowflakeSourceAccessor snowflakeAccessor, String
     }
   }
 
+  public static Schema getParsedSchema(String schema) {
+    try {
```

Comment:
Add null check here as schema can be null, in that case it should return null

Reply:
And no need to have schema as input parameter for the method

Reply:
and make the method private

Reply:
Updated

Reply:
> And no need to have schema as input parameter for the method

Didn't understand this request

Reply:
> > And no need to have schema as input parameter for the method
> 
> Didn't understand this request

ignore it. I thought it's a class level variable

Reply:
squash the commits and raise an external PR